app/test_main.py

```diff
@@ -1 +1,42 @@
-# write your code here
+import pytest
+
+from unittest import mock
+
+from app.main import can_access_google_page
+
+
+@pytest.mark.parametrize(
+    "current_time,current_url,exception_message",
+    [
+        (
+            True,
+            True,
+            "Accessible"
+        ),
+        (
+            False,
+            True,
+            "Not accessible"
+        ),
+        (
+            True,
+            False,
+            "Not accessible"
+        ),
+        (
+            False,
+            False,
+            "Not accessible"
+        )
+    ]
+)
+def test_can_access_to_page(
+        current_time: bool,
+        current_url: bool,
+        exception_message: str
+) -> None:
+    with mock.patch("app.main.valid_google_url") as mock_url:
+        with mock.patch("app.main.has_internet_connection") as mock_connection:
```

Comment:
Non needed two with keywords. You can use it:
`with A() as a, B() as b:`

Reply:
Ok, thank you!